cpp/iedriver/IECommandExecutor.cpp

```diff
@@ -465,6 +465,13 @@ LRESULT IECommandExecutor::OnBrowserQuit(UINT uMsg,
   } else {
     LOG(WARN) << "Unable to find browser to quit with ID " << browser_id;
   }
+
+  ::Sleep(3000);
```

Comment:
If we really need to sleep for three full seconds, let’s document that. If the reason for that duration is arbitrary, comment it so. If there’s something else we can poll on to make sure it’s safe to delete the directory, let’s do that instead.

Reply:
Updated the sleep 3000ms to sleep 100ms with retries. It usually completes in 0 or 1 retries in my local tests.